Makefile

```diff
@@ -34,10 +34,11 @@ generate-mocks:
 	@echo making sure directory for mocks exists
 	mkdir -p $(MOCKS_DIR)
 
+	# $(mockgen) -destination=<Path/file where the mock is generated> -package=<The package that the generated mock files will belong to> -mock_names=<Original Interface name>=<Name of Generated mocked Interface> <Go package path of the original interface> <comma seperated list of the interface you want to mock>
```

Comment:
Why add this comment?

Reply:
@kevinrizza I had included this to help me understand/remember the meaning of the command. Thought it would be useful for future contributors who are new to `mock`.

Reply:
It just seems unrelated to the rest of the pr. If you feel that this comment is useful, my suggestion is to create a separate pr for it.

Reply:
Well he is changing a line in this file, so I am fine allowing it. @tkashem can be the tie breaker.

Reply:
A related comment is being added to the very thing worked on. It looks good to me.